plugins/content/tjreportsfields/tjreportsfields.php

```diff
@@ -0,0 +1,289 @@
+<?php
+/**
+ * @package     TJReports
+ * @subpackage  com_tjreports.tjreportsfields
+ *
+ * @copyright   Copyright (C) 2009 - 2019 Techjoomla. All rights reserved.
+ * @license     http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * Class for Tjreportsfields Content Plugin
+ *
+ * @since  1.1.0
+ */
+class PlgContentTjreportsfields extends JPlugin
+{
+	/**
+	 * Load the language file on instantiation.
+	 *
+	 * @var    boolean
+	 *
+	 * @since  3.2.11
+	 */
+	protected $autoloadLanguage = true;
+
+	protected $customFieldsTable;
+
+	protected $customFieldBeingEdited;
+
+	protected $unSupportedFields = array ('imagelist', 'media', 'repeatable', 'sql');
```

Comment:
sql field outputs a dropdown right ? Why can't we support it ?

Reply:
This outputs dropdown. For other types it is easy to get a dropdown text from value, for this type, there is no easy way.
Let's add support in the future if needed